frontend/packages/noobaa-storage-plugin/src/components/noobaa-operator/infoPage.scss

```diff
@@ -0,0 +1,35 @@
+.nb-info__section {
```

Comment:
ident is off, somethimes its 4 spaces instead of 2

Reply:
fixed